packages/forms-web-app/src/routes/full-appeal/index.js

```diff
@@ -13,6 +13,11 @@ router.use(require('./login'));
 /// as well as appeal objects
 router.use('/submit-final-comment', require('./submit-final-comment'));
 
+router.use(
+	checkAppealExists,
+	checkDecisionDateDeadline,
+	require('./submit-appeal/planning-application-number')
+);
```

Comment:
I take it this one doesn't need to be logged in to access?

Reply:
Yes it goes:
- before you start
- planning application number
- email